docs/public/specifications/tests/DNSSEC-TP/dnssec18.md

```diff
@@ -216,13 +216,14 @@ None.
 [DS18_NO_MATCH_CDS_RRSIG_DS]:            #summary
 [Default level]:                         ../SeverityLevelDefinitions.md
 [ERROR]:                                 ../SeverityLevelDefinitions.md#error
+[Get-Parent-NS-IP]:                      ../MethodsV2.md#method-get-parent-ns-ip-addresses
 [INFO]:                                  ../SeverityLevelDefinitions.md#info
 [Method1]:                               ../Methods.md#method-1-obtain-the-parent-domain
 [Method4]:                               ../Methods.md#method-4-obtain-glue-address-records-from-parent
 [Method5]:                               ../Methods.md#method-5-obtain-the-name-server-address-records-from-child
 [NOTICE]:                                ../SeverityLevelDefinitions.md#notice
 [RFC 4035#2.4]:                          https://datatracker.ietf.org/doc/html/rfc4035#section-2.4
 [RFC 7344#4.1]:                          https://datatracker.ietf.org/doc/html/rfc7344#section-4.1
-[RFC 7344]:                              https://datatracker.ietf.org/doc/html/rfc7344
-[RFC 8078]:                              https://datatracker.ietf.org/doc/html/rfc8078
+[RFC 7344]: https://datatracker.ietf.org/doc/html/rfc7344
+[RFC 8078]: https://datatracker.ietf.org/doc/html/rfc8078
```

Comment:
I suggests that the indentation is kept. I makes it easier to read.

Reply:
Fixed.